test/index.html

```diff
@@ -39,12 +39,6 @@
     // assertion below
     require('@glimmer/util');
 
-    Object.assign = function () {
```

Comment:
Tests are red locally without this change: https://github.com/glimmerjs/glimmer-vm/pull/1413